pkg/apiserver/api_installer.go

```diff
@@ -108,6 +108,7 @@ func (a *APIInstaller) NewWebService() *restful.WebService {
 	mediaTypes, streamMediaTypes := mediaTypesForSerializer(a.group.Serializer)
 	ws.Produces(append(mediaTypes, streamMediaTypes...)...)
 	ws.ApiVersion(a.group.GroupVersion.String())
+	ws.SetDynamicRoutes(true)
```

Comment:
> @liggitt this attribute is just webservice level, which gives us chance to remove routes handler of a specific webservice. I am not quite clear what you mean nested web service were allowed to be removed, would you mind explain more?

Can you create a webservice with a narrower scope for each TPR group?  This sort of change is why we're interested in breaking TPRs out of the main API server.